content/en/real_user_monitoring/faq/session_replay_service_worker.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Allow Third-Party Service Worker for Session Replay
+kind: faq
+further_reading:
+    - link: '/real_user_monitoring/session_replay/'
+      tag: 'Get Started'
+      text: 'Session Replay'
+---
+
+## Third-Party Cookie Blocking And Session Replay
+
+Session Replay uses a *Service Worker* in a different domain (*session-replay-datadoghq.com*) to provide the best experience possible whilst protecting your privacy and guaranteeing the safety of your data.
+
+If you have blocked third-party cookies in your browser settings, or if your browser blocks them by default, this can prevent the service worker from registering correctly.
```

Comment:
```suggestion
If you have blocked third-party cookies in your browser settings, or if your browser blocks them by default, this prevents the service worker from registering your replays correctly.
```

Reply:
... from registering **your replays** correctly.

It isn't registering _replays_, but it's the registration of the Service Worker itself. Just this change is not technically accurate.

Reply:
Thanks for the additional feedback!